src/comments/dto/comment-create.dto.ts

```diff
@@ -0,0 +1,21 @@
+import { Comment } from '@prisma/client';
+import { IsString, IsDefined, IsEmpty } from 'class-validator';
+
+export class CommentCreateDto implements Omit<Comment, 'authorID' | 'id'> {
+	@IsDefined()
+	@IsString()
+	commentBody: string;
+
+	@IsDefined()
+	@IsString()
+	articleID: string;
```

Comment:
use custom middleware please for checking ids

Reply:
Done